src/lib/chips/chip-list.ts

```diff
@@ -49,69 +49,125 @@ import {SPACE, LEFT_ARROW, RIGHT_ARROW} from '../core/keyboard/keycodes';
 })
 export class MdChipList implements AfterContentInit {
 
-  /** Track which chips we're listening to for focus/destruction. */
-  private _subscribed: WeakMap<MdChip, boolean> = new WeakMap();
+  /** Whether or not the chip list is currently focusable via keyboard interaction. */
```

Comment:
This isn't a boolean, so `Whether...` isn't really the right way to phrase it. I think it's fine to say "tabIndex of the chip-list element."